core/domain/user_id_migration.py

```diff
@@ -117,6 +118,9 @@ def _change_model_with_one_user_id_field(
             migration_field == old_user_id).fetch()
         for model in found_models:
             model_values = model.to_dict()
+            # We need to get the string name of the field here and in order to
```

Comment:
Hi @vojtechjelinek -- this is a run-on sentence. I've read it three times and I still can't understand it. Could you please simplify it and maybe break it down into smaller sentences to help with comprehension?

Reply:
Done.